packages/auth/src/pubKeys/index.ts

```diff
@@ -0,0 +1,17 @@
+import { keys as prodPubKeys } from './auth0-pubkeys.prod.json';
+import { keys as devPubKeys } from './auth0-pubkeys.dev.json';
+
+interface JWK {
```

Comment:
Just define them as TS instead of JSON I'd say, should save you from manually maintaining typings

Reply:
I would rather just have to `wget` the file if it needs to be updated. I want to keep it in the default format

Reply:
Hmm, but it's literally prepending `export default` isn't it, the formatter does the rest

Reply:
Yes, I would rather not have to prepend anything. Plus, the type is simple enough that I don't mind defining it.